### PR TITLE
Restaurant features

### PR DESCRIPTION
**Pull request by edwinmoradian90**

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="../src/assets/css/style.css">
+    <title>Restaurant Page</title>
+</head>
+<body>
+    <div id="content">
+    </div>
+    <script type="module" src="main.js"></script>
```
Comment (azdrenymeri) [inline dist/index.html]: When adding the `type="module"`causes the code to be treated as a JavaScript module.  Unlike classic scripts, module scripts require the use of the CORS protocol for cross-origin fetching. And chrome will not allow loading of this script throwing an error:
scr
### Kindly remove this attribute from your script tag.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash';
```
Comment (azdrenymeri) [inline src/index.js]: This import(lodash) is imported but never used.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        setActive(menuTab);
+    } else
+    if(click.matches('.contact')){
+        removeSection();
+        section.appendChild(contact());
+        setActive(contactTab);
+    }
+})
+
+console.log('working')
```
Comment (azdrenymeri) [inline src/index.js]: Kindly remove this console.log statement

File: dist/index.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="../src/assets/css/style.css">
```
Comment (azdrenymeri) [inline dist/index.html]: You should use `style-loaders` webpack plugin to import your stylesheets.
Kindly read this carefully and follow the steps: https://webpack.js.org/concepts/loaders/ on how to add style loaders on your webpack project.

File: README.md [hunk 5]
```diff
@@ -1 +1,20 @@
...
+The focus of the project was to get used to node modules.
+
+## Installation
+
+Just clone and run the following code.
+
+```bash
+npm install
+```
+
```
Comment (shloch) [inline README.md:14]: okease detail all the commands for installing/settup project locally,

**Pull request merged**